### PR TITLE
Isolate Header Banner

### PR DESCRIPTION
**Pull request by Osvaldoav**
### What does this PR do?

Creates a reusable Header Banner component and deletes other specific Header Components:
- Members
- Contact
- Candidates
- Projects

#### Related issues
#63 

**Todo:** Change icon button color scheme

### CODE REVIEW

File: src/components/Projects/Projects.test.js [hunk 1]
```diff
@@ -67,7 +66,6 @@ beforeEach(() => {
 
 it('<Projects> render main components correctly', () => {
   expect(wrapper.find(ProjectsComponent).length).toBe(1)
-  expect(wrapper.find(ProjectsHeader).length).toBe(1)
```
Comment (AuroTB) [inline src/components/Projects/Projects.test.js:70]: we should validate the existance of the header, please add it to the test

File: src/components/Shared/HeaderBanner/HeaderBanner.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    right: 0;
+    margin-left:auto;
+    margin-right:auto;
+  }
+  
+  .main-container .footer p {
+    margin: 0;
+  }
+  
+  .main-container .footer .icon-btn {
```
Comment (AuroTB) [inline src/components/Shared/HeaderBanner/HeaderBanner.css:40]: Delete icon-btn class styling, as well as its hover plz

File: src/components/Shared/HeaderBanner/HeaderBanner.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  const parseText = (text) => text.map((textLine) => (
+    <>
+      { textLine }
+      <br />
+    </>
+  ))
+
+  return (
+    <div className="main-container" style={styles.mainContainer}>
+      <div className="container-legend">
```
Comment (AuroTB) [inline src/components/Shared/HeaderBanner/HeaderBanner.js:59]: Add the styling of this class (container-legend) in the .css
You can find it in src>index.css (also, delete it from that file, plz)

File: src/components/Shared/HeaderBanner/HeaderBanner.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    bgColorScheme: ColorScheme,
+    iconColorScheme: ColorScheme
+};
+
+function HeaderBanner({
+  title, mainText, subText, bgColorScheme, iconColorScheme,
+}: Props) {
+  const [iconColor, setIconColor] = useState(iconColorScheme.primary)
+
+  const styles = {
```
Comment (AuroTB) [inline src/components/Shared/HeaderBanner/HeaderBanner.js:26]: nice

File: src/components/Shared/HeaderBanner/HeaderBanner.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  */
+  const scrollToInfo = () => {
+    window.scrollBy(0, window.innerHeight - 48 - window.scrollY)
+  }
+
+  /**
+   * Function to parse array of strings into lines of text in jsx using <br/> tags.
+   * @param {text} text: Array of strings, representing each line of text.
+   */
+  const parseText = (text) => text.map((textLine) => (
```
Comment (AuroTB) [inline src/components/Shared/HeaderBanner/HeaderBanner.js:50]: so good

File: src/components/Shared/HeaderBanner/HeaderBanner.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+type Props = {
+    title: string,
+    mainText: Array<string>,
+    subText: Array<string>,
+    bgColorScheme: ColorScheme,
+    iconColorScheme: ColorScheme
+};
+
+function HeaderBanner({
```
Comment (AuroTB) [inline src/components/Shared/HeaderBanner/HeaderBanner.js:21]: nice, maybe we should change some classes that don't do much to functional components

**Review by AuroTB — CHANGES_REQUESTED** [review]

**Comment by AuroTB** [comment]
Btw, you can copy the unit test already made from the previous [proyects header test](https://github.com/RoBorregos/roborregos-web/blob/develop/src/components/Projects/ProjectsHeader/ProjectsHeader.test.js).

It includes scrolling and component rendering, just modify it with the corresponding components.

File: src/components/Shared/HeaderBanner/HeaderBanner.css (hunk 2 above)
Comment (Osvaldoav) [reply src/components/Shared/HeaderBanner/HeaderBanner.css:49]: I left the .icon-btn class and removed only the color property instead. Now the .js file manage the color of the icon.

File: src/components/Shared/HeaderBanner/HeaderBanner.js (hunk 6 above)
Comment (Osvaldoav) [reply src/components/Shared/HeaderBanner/HeaderBanner.js:21]: +1

**Review by AuroTB — APPROVED** [review]

**Pull request merged**